Standalone jar requires net connection even for local GeoPackage

The standalone jar file cannot be used for testing a local GeoPackage with a default test-run-props.xml file if there is no network connection. The reason is this reference in the props file
`<!DOCTYPE properties SYSTEM "http://java.sun.com/dtd/properties.dtd">`

Offline check succeeds by downloading the DTD into local file system and changing the reference
`<!DOCTYPE properties SYSTEM "file:///C:/teamengine/properties.dtd">`

As an enhancement I suggest to document this workaround if it is not reasonable to add a copy of properties.dtd into the jar file or to skip the DTD check.

Following is my workaround:
[1] `<!DOCTYPE properties SYSTEM "http://java.sun.com/dtd/properties.dtd">`
		
[2] `<!DOCTYPE properties SYSTEM "C:\properties.dtd">`

I am unable to build test-suite normally with reference [2] in test-run-props.xml because JUnit test cases are getting failed with below exception: 
```
org.opengis.cite.gpkg12.VerifyTestNGController
cleanTestRun(org.opengis.cite.gpkg12.VerifyTestNGController)  Time elapsed: 0.013 sec  <<< ERROR!
java.util.InvalidPropertiesFormatException: org.xml.sax.SAXException: Invalid system identifier: file:///C:/properties.dtd
        at org.opengis.cite.gpkg12.VerifyTestNGController.loadDefaultTestRunProperties(VerifyTestNGController.java:50)
Caused by: org.xml.sax.SAXException: Invalid system identifier: file:///C:/properties.dtd
        at org.opengis.cite.gpkg12.VerifyTestNGController.loadDefaultTestRunProperties(VerifyTestNGController.java:50)
```
The reason of above failure is loadXMLFrom method of Properties class in the following JUnit test class https://github.com/opengeospatial/ets-gpkg12/blob/eeb9663e164705ed1675bbca586cd9d23af00f4f/src/test/java/org/opengis/cite/gpkg12/VerifyTestNGController.java#L50 which expects the XML document must have the following DOCTYPE declaration: 
`<!DOCTYPE properties SYSTEM "http://java.sun.com/dtd/properties.dtd">`

I am able to build the test-suite by skipping JUnit test cases and executed successfully.

--- a/src/main/resources/org/opengis/cite/gpkg12/dtd/properties.dtd
+++ b/src/main/resources/org/opengis/cite/gpkg12/dtd/properties.dtd
@@ -1,0 +1,15 @@
+<!--
+   Copyright 2006 Sun Microsystems, Inc.  All rights reserved.
+  -->
+
+<!-- DTD for properties -->
+
+<!ELEMENT properties ( comment?, entry* ) >
+
+<!ATTLIST properties version CDATA #FIXED "1.0">
+
+<!ELEMENT comment (#PCDATA) >
+
+<!ELEMENT entry (#PCDATA) >
+
+<!ATTLIST entry key CDATA #REQUIRED>

--- a/src/main/resources/test-run-props.xml
+++ b/src/main/resources/test-run-props.xml
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE properties SYSTEM "http://java.sun.com/dtd/properties.dtd">
+<!DOCTYPE root-name SYSTEM "./org/opengis/cite/gpkg12/dtd/properties.dtd">
 <properties version="1.0">
     <comment>Test run arguments</comment>
     <entry key="iut">http://www.geopackage.org/data/simple_sewer_features.gpkg</entry>

--- a/src/test/java/org/opengis/cite/gpkg12/VerifyTestNGController.java
+++ b/src/test/java/org/opengis/cite/gpkg12/VerifyTestNGController.java
@@ -47,7 +47,7 @@ public class VerifyTestNGController {
     @Before
     public void loadDefaultTestRunProperties() throws InvalidPropertiesFormatException, IOException {
         this.testRunProps = new Properties();
-        this.testRunProps.loadFromXML(getClass().getResourceAsStream("/test-run-props.xml"));
+        this.testRunProps.load(getClass().getResourceAsStream("/test-run-props.xml"));
     }
 
     @Test

